addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -1741,6 +1741,61 @@ options.registry.collapse = options.Class.extend({
     },
 });
 
+options.registry.masonry_display = options.Class.extend({
```

Comment:
New option names should follow the convention for classes (PascalCase aka UpperCamelCase) => MasonryLayout or some such.